src/pages/admin/index.js

```diff
@@ -0,0 +1,127 @@
+    TableBody,
+    TableCell,
+    TableContainer,
+    TableHead,
+    TableRow,
+    Paper
+} from "@material-ui/core";
+import { baseUrl } from "../../utils/baseUrl";
+
+const StyledTableCell = withStyles((theme) => ({
+    head: {
+        backgroundColor: theme.palette.secondary.main,
+        color: theme.palette.common.white,
+    },
+    body: {
+        fontSize: 14,
+    },
+}))(TableCell);
+
+const StyledTableRow = withStyles((theme) => ({
+    root: {
+        '&:nth-of-type(odd)': {
+            backgroundColor: theme.palette.action.hover,
+        },
+    },
+}))(TableRow);
+
+const useStyles = makeStyles((theme) => ({
+    toolbar: {
+        display: "flex",
+        justifyContent: "space-between",
+    },
+    container: {
+        marginTop: theme.spacing(10),
+    },
+    table: {
+        marginTop: theme.spacing(2),
+    },
+    link: {
+        textDecoration: "none",
+        color: theme.palette.info.main,
+    },
+    errorMessage: {
+        color: theme.palette.error.main,
+        margin: 0,
+    },
+}));
+
+export default function Admin() {
+    const classes = useStyles();
+
+    const [users, setUsers] = useState([]);
+
+    useEffect(() => {
+        axios
+            .get(`${baseUrl}/admin`)
+            .then((res) => {
+                setUsers(res.data.users)
+            })
+            .catch((err) => {
+                console.error(err);
+            });
+    }, [users]);
+
+    return (
+        <>
+            <Head>
+                <title>Admin Page</title>
+                <meta
+                    name="description"
+                    content="Admin Page - Todo App dashboard"
+                />
+                <link rel="icon" href="/favicon.ico" />
+            </Head>
+            <body>
+                <AppBar>
+                    <Toolbar className={classes.toolbar}>
+                        <Typography component="h1" variant="h5">
+                            Admin Page
+                        </Typography>
+                        <Button color="inherit">Log out</Button>
+                    </Toolbar>
+                </AppBar>
+                <Container component="main" className={classes.container}>
+                    <Typography component="h1" variant="h6">
+                        Registered User
+                    </Typography>
+                    <TableContainer component={Paper} className={classes.table}>
+                        <Table >
+                            <TableHead>
+                                <TableRow>
+                                    <StyledTableCell>Email</StyledTableCell>
+                                    <StyledTableCell align="left">Role</StyledTableCell>
+                                    <StyledTableCell align="left">Is Active</StyledTableCell>
+                                </TableRow>
+                            </TableHead>
+                            <TableBody>
+                                {users.map((user) => (
```

Comment:
When I tried it, the users is not showed, can you investigate it?

Reply:
Seems like the API not running in Netlify mas, I try it in the local, it works perfectly

Reply:
Yes it because you call hardcoded localhost:3000 when deployed.

How to debug:
1. Open inspect element
2. Click network tab
3. Try when you click login button and watch the api request list on Network tab
4. It will call localhost:3000 which is not found url address

https://user-images.githubusercontent.com/22090926/132224638-4891ff86-6b20-423e-9665-55d16fe1e404.mov

Reply:
Should I change it to the domain that Netlify provides for this app?

Reply:
yes, you can check the discord for another way